ParabolInc/parabol
# fix(chore): properly count emojis length

# Description

Fixes #8098 

## Demo

<img width="851" alt="Screenshot 2023-04-27 at 14 47 04" src="https://user-images.githubusercontent.com/1879975/234784851-aa40fa74-daef-4c79-8a1b-73f81ca9003a.png">


## Testing scenarios

- [ ] Emojis
  - Create a new Poker Scale
  - Add a new value 🐈‍⬛
  - Verify there is no client or server side error

- [ ] Allow max 3 characters
  - Create a new Poker Scale
  - Add a new value XXS
  - Verify there is no client or server side error

## Final checklist
